app/models/concerns/foreman_remote_execution/user_extensions.rb

```diff
@@ -0,0 +1,9 @@
+module ForemanRemoteExecution
+  module UserExtensions
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :targetings, :dependent => :nullify
```

Comment:
I assume you mean tying targetings to user groups rather than users?  I might could see that as an enhancement, but its not in the initial design